applications/GeoMechanicsApplication/custom_elements/transient_Pw_line_element.h

```diff
@@ -281,9 +281,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientPwLineElement : public Elem
     {
         const auto number_integration_points = GetGeometry().IntegrationPointsNumber(GetIntegrationMethod());
         GeometryType::JacobiansType J_container{number_integration_points};
-        for (std::size_t i = 0; i < number_integration_points; ++i) {
-            J_container[i].resize(GetGeometry().WorkingSpaceDimension(),
-                                  GetGeometry().LocalSpaceDimension(), false);
+        for (auto& container : J_container) {
```

Comment:
If I'm correct in other parts of the code this is called `j`:
```suggestion
        for (auto& j : J_container) {
```

Reply:
Done